README.md

```diff
@@ -110,41 +110,42 @@ your distribution's package repository (Debian calls them `liblocal-lib-perl`
 and `cpanminus` respectively). `cpanm` depends on the `make` and `gcc` commands
 being available.
 
-NOTE: gcc can be removed after all cpan dependencies are installed.
+NOTE: `gcc` can be removed after all `cpan` dependencies are installed.
       However, it may be necessary to reinstall it if additional modules are
       required during an upgrade
 
-To install the Perl module dependencies, run:
+To install the `Perl` module dependencies, run:
 
 ```sh
  cpanm --quiet --notest --with-feature=starman [other features] --installdeps .
 ```
 
-NOTE: Don't miss the "." at the end of the cpanm command!
+NOTE: Don't miss the "." at the end of the `cpanm` command!
 Don't forget to make sure the environment variable
-`PERL5LIB=/home/ledgersmb/perl5/lib/perl5` points at the running user's perl5 dir
-Also, NEVER run cpanm as root, it's best to run it as the user you intend to
-run ledgersmb as when possible. This installs the cpan modules in `~/perl5`
+`PERL5LIB=/home/ledgersmb/perl5/lib/perl5` points at the running user's `perl5` dir
+Also, NEVER run `cpanm` as root, it's best to run it as the user you intend to
+run ledgersmb as when possible. This installs the `cpan` modules in `~/perl5`
+
 If you can't run it as the final user, don't worry, just run it as any
-user (eg: johnny), and make sure the environment variable
-`PERL5LIB=/home/johhny/perl5/lib/perl5` points at jonny's perl5 dir
+user (e.g.: johnny), and make sure the environment variable
+`PERL5LIB=/home/johhny/perl5/lib/perl5` points at johnny's `perl5` directory
```

Comment:
Spelling of `johnny` is still inconsistent.
variations are
- johnny
- johhny

Reply:
Fixed